packages/flutter/lib/fix_data.yaml

```diff
@@ -11,6 +11,149 @@
 version: 1
 transforms:
 
+  # Changes made in https://github.com/flutter/flutter/pull/44189.
+  - title: 'Rename to dependOnInheritedElement'
+    date: 2020-12-23
+    element:
+      uris: [ 'material.dart' ]
```

Comment:
dependOnInheritedElement is defined on Element, which is exported by widgets.dart. DOesn't this list need to include widgets.dart (and other libraries that export it, e.g. Cupertino?)

Reply:
Same below.